.github/workflows/changelog.yml

```diff
@@ -0,0 +1,19 @@
+name: bandersnatch_changelog
+
+on:
+  pull_request:
+    types: [opened, synchronize, labeled, unlabeled, reopened]
+
+jobs:
+  build:
+    name: Changelog Entry Check
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
+
+    - name: Grep CHANGES.md for PR number
+      if: contains(github.event.pull_request.labels.*.name, 'skip news') != true
+      run: |
+        grep -P "PR #${{ github.event.pull_request.number }}[^0-9]" CHANGES.md
```

Comment:
Wow, much better and fancier regex that I was gonna do ! <3

Reply:
admittingly this was me over worrying grep matching a toally unrelated PR, having thought it through more, the situation I was worrying about seems impossible, but oh well, it's alright

Reply:
I'm a forward fix and done is better than perfect kind of person :) We can iterate.